inventory/README.md

```diff
@@ -130,523 +151,131 @@ To build the application, we used maven build. Maven is a project management too
 ```
 [INFO] --- maven-failsafe-plugin:2.18.1:verify (verify-results) @ inventory ---
 [INFO] Failsafe report directory: /Users/user@ibm.com/BlueCompute/refarch-cloudnative-micro-inventory/inventory/target/test-reports/it
-[INFO] 
+[INFO]
 [INFO] --- maven-install-plugin:2.4:install (default-install) @ inventory ---
 [INFO] Installing /Users/user@ibm.com/BlueCompute/refarch-cloudnative-micro-inventory/inventory/target/inventory-1.0-SNAPSHOT.war to /Users/user@ibm.com/.m2/repository/projects/inventory/1.0-SNAPSHOT/inventory-1.0-SNAPSHOT.war
 [INFO] Installing /Users/user@ibm.com/BlueCompute/refarch-cloudnative-micro-inventory/inventory/pom.xml to /Users/user@ibm.com/.m2/repository/projects/inventory/1.0-SNAPSHOT/inventory-1.0-SNAPSHOT.pom
 [INFO] ------------------------------------------------------------------------
 [INFO] BUILD SUCCESS
 [INFO] ------------------------------------------------------------------------
-[INFO] Total time: 39.004 s
-[INFO] Finished at: 2018-02-22T15:14:59-06:00
-[INFO] Final Memory: 22M/307M
+[INFO] Total time: 22.516 s
+[INFO] Finished at: 2018-07-18T14:24:10-05:00
+[INFO] Final Memory: 32M/467M
 [INFO] ------------------------------------------------------------------------
 ```
 ### Setting up MYSQL
```

Comment:
MYSQL should be MySQL?

Reply:
Modified to `MySQL`. Changed the sequence ... IBM Cloud Private and then comes Minikube